### PR TITLE
feat: filter notes

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-13

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- code looks neat and clean
- I like you put all helper function in utils folder 
- good to merge

File: note-app/src/Utils/SortByDate.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+export const SortByDate=(note,sortByDate)=>{
+   switch(sortByDate){
+       case "old-to-new":
+       return [...note].sort((a,b) => new Date (a.CreatedAt) - new Date (b.CreatedAt))
```
Comment (SanketDhabarde) [inline note-app/src/Utils/SortByDate.js:5]: Like this that you kept it pure

File: note-app/src/Utils/Priority.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+export const Priority=(note,priority)=>{
+     if(priority.length !==0){
```
Comment (SanketDhabarde) [inline note-app/src/Utils/Priority.js]: you can directly put `priority.length` if it's falsy value like 0 it will go to else block

**Review by SanketDhabarde — APPROVED** [review]
- Implementation looks proper
- Code structure is good
- good to merge

File: note-app/src/Utils/Priority.js (hunk 2 above)
Comment (aniketsingh13) [reply note-app/src/Utils/Priority.js]: ok,will do that

**Pull request merged**